src/NuGet.Core/NuGet.Build.Tasks.Pack.Library/Pack.targets

```diff
@@ -68,20 +68,7 @@ Copyright (c) .NET Foundation. All rights reserved.
     <Delete Files="@(_PackageFilesToDelete)"/>
   </Target>
 
-  <!--
-    ============================================================
-    Pack
-    Main entry point for packing packages
-    ============================================================
-  -->
-  <Target Name="Pack" DependsOnTargets="_IntermediatePack;$(PackDependsOn)">
-  </Target>
-  <Target Name="_IntermediatePack">
-    <PropertyGroup>
-      <ContinuePackingAfterGeneratingNuspec>true</ContinuePackingAfterGeneratingNuspec>
-    </PropertyGroup>
-  </Target>
-  <Target Name="GenerateNuspec" DependsOnTargets="$(GenerateNuspecDependsOn)" Condition="$(IsPackable) == 'true'">
+  <Target Name="_CaclulateInputsOutputsForPack">
```

Comment:
Typo: `Ca`**`cl`**`ulateInputsOutputsForPack`. Don't forget to change the reference below too.